deploy.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -ev
+echo `zip -r extension.zip extension/`
+echo `grunt`
```

Comment:
I'm following the Travis documentation. The echo in back ticks executes the command.